Tests/PointFreeTests/PaymentInfoTests.swift

```diff
@@ -0,0 +1,44 @@
+import Either
+import Html
+import HtmlPrettyPrint
+import HttpPipeline
+@testable import PointFree
+import PointFreeTestSupport
+import Prelude
+import Optics
+import SnapshotTesting
+import XCTest
+#if !os(Linux)
+  import WebKit
+#endif
+
+class PaymentInfoTests: TestCase {
+  override func setUp() {
+    super.setUp()
+    AppEnvironment.push(\.database .~ .mock)
+  }
+
+  override func tearDown() {
+    super.tearDown()
+    AppEnvironment.pop()
+  }
+
+  func testRender() {
+    let conn = connection(from: request(to: url(to: .paymentInfo)))
+    let result = conn |> siteMiddleware
+
+    assertSnapshot(matching: result.perform())
+
+    #if !os(Linux)
+      if #available(OSX 10.13, *) {
+        let webView = WKWebView(frame: .init(x: 0, y: 0, width: 1080, height: 2000))
```

Comment:
these screenshots seem to be much smaller than the others, so could cut the size of em down

Reply:
Yeah, this view isn't ready yet. Will definitely fix the height when I finish up tho.